### ISSUE TITLE
srcset processed incorrectly when comma in URL

### ISSUE THREAD

**Issue by dcastil** [vuejs/vuepress#990]
- [x] I confirm that this is a issue rather than a question.

#### Version
`vuepress@1.0.0-alpha.19` and `vuepress@0.14.4`

#### Steps to reproduce

```bash
mkdir test && cd test
yarn global add vuepress@1.0.0-alpha.19
echo '<img src="https://url.com/with,comma" srcset="https://url.com/with,comma_200 200w, https://url.com/with,comma_400 400w">' > README.md
vuepress build
```

#### What is expected?

in `.vuepress/dist/index.html` an image like this should be rendered.

```html
<img
  src="https://url.com/with,comma"
  srcset="https://url.com/with,comma_200 200w, https://url.com/with,comma_400 400w"
>
```

#### What is actually happening?

This image is being rendered. Note the space inserted after every comma.

```html
<img
  src="https://url.com/with,comma"
  srcset="https://url.com/with, comma_200 200w, https://url.com/with, comma_400 400w"
>
```

The browser can't interpret this image correctly anymore.

#### Other relevant information

- Your OS: Mac OS Mojave 10.14.1
- Node.js version: 10.11.0
- Browser version: Chrome 70.0.3538.77
- Is this a global or local install? local
- Which package manager did you use for the install? yarn

I ran into this problem while working on enhancing markdown-processing of images stored on Cloudinary. The Cloudinary URLs often contain commas for image transformations.

**Comment by ulivz** [comment]
Thanks for your interest in VuePress, but this issue should be moved to [markdown-it](https://github.com/markdown-it/markdown-it).

FYI: Maybe this change would not accepted by markdown-it, see [this article](https://www.searchenginenews.com/sample/content/are-you-using-commas-in-your-urls-heres-what-you-need-to-know).

**Comment by dcastil** [comment]
@ulivz That's not an issue with markdown-it, I already tested it. Markdown-it returns the correct value and the URL is changed somewhere afterwards. Here is what markdown-it returns:

```
<p>&lt;img src=&quot;https://url.com/with,comma&quot; srcset=&quot;https://url.com/with,comma_200 200w, https://url.com/with,comma_400 400w&quot;&gt;</p>
```

I'd debug this by myself but I find it really hard to follow the data through the process. I just know that it is already broken in line 147 in `packages/@vuepress/core/lib/build.js` where it's assigned to the `html` variable.

**Comment by finex** [comment]
Hi, I'm experiencing this exact bug on nuxt.js with cloudinary image paths inside the srcset attribute.
